modules/users/client/components/InterfaceLanguagePanel.component.js

```diff
@@ -32,7 +32,9 @@ export default function InterfaceLanguagePanel() {
                   {t(
                     'Thanks to all our community members who helped translate!',
                   )}{' '}
-                  <a href="/volunteering">{t('You can help us out!')}</a>
+                  <a href="https://openhospitality.network/">
```

Comment:
My preference would be to also put this in a constants file somewhere, so you can change a bit more of the branding-specific stuff in one go

Reply:
Yep, I agree